src/index.css

```diff
@@ -1,5 +1,64 @@
+@font-face {
+  font-family: 'Roboto Regular';
+  src: url('/font/Roboto-Regular.tff');
```

Comment:
We have to take care to not load too many fonts (3G network)
(just a warning)

Reply:
it's not the font downloading which slows the load.
I add a rule to use local font if exists instead of download it